src/function/table/system/pragma_table_info.cpp

```diff
@@ -111,7 +111,11 @@ static void PragmaTableInfoTable(PragmaTableOperatorData &data, TableCatalogEntr
 		// "name", PhysicalType::VARCHAR
 		output.SetValue(1, index, Value(column.Name()));
 		// "type", PhysicalType::VARCHAR
-		output.SetValue(2, index, Value(column.Type().ToString()));
+		if (column.Type().id() == LogicalTypeId::ENUM) {
+			output.SetValue(2, index, Value("ENUM")); // overwrites typename for compatibility with SQLAlchemy.See #5174
```

Comment:
nit: Can you use `LogicalTypeIdToString` instead? slightly less hardcoded that way :P